server/v2/stf/stf.go

```diff
@@ -17,31 +17,10 @@ import (
 	stfgas "cosmossdk.io/server/v2/stf/gas"
 )
 
-var _ STF[transaction.Tx] = STF[transaction.Tx]{} // Ensure STF implements STFI.
-
-// STFI defines the state transition handler used by AppManager to execute
-// state transitions over some state. STF never writes to state, instead
-// returns the state changes caused by the state transitions.
-type STFI[T transaction.Tx] interface {
```

Comment:
this should probably be defined in appamanger instead of here. maybe thats what @testinginprod meant?